spokestack/asr/speech_recognizer.py

```diff
@@ -80,7 +80,9 @@ def _receive(self, context):
             hypothesis = hypotheses[0]
             context.transcript = hypothesis["transcript"]
             context.confidence = hypothesis["confidence"]
-        self._is_final = self._client.response.get("final")
+            context.event("partial_recognize")
+        if self._client.response.get("final"):
+            context.event("recognize")
```

Comment:
Good thing I read my GitHub emails...

Reply:
I think we talked about this in slack a while back, so I figured a fix was in progress
I only mention it here because will's port was correct, but the original was wrong

Reply:
https://spokestack.slack.com/archives/CHHAM1519/p1600981282041800